src/OracleService/Protocols/OracleHttpsProtocol.cs

```diff
@@ -22,7 +22,11 @@ namespace Neo.Plugins
 {
     class OracleHttpsProtocol : IOracleProtocol
     {
-        private readonly HttpClient client = new(new HttpClientHandler() { AllowAutoRedirect = false });
+        private readonly HttpClient client = new(new HttpClientHandler() { AllowAutoRedirect = false })
+        {
+            Timeout = TimeSpan.FromSeconds(30),
+            MaxResponseContentBufferSize = ushort.MaxValue
```

Comment:
Don't we want to make this value configurable?

Reply:
It's `OracleResponse.MaxResultSize` effectively. Maybe this constant can be used instead.